py4cast/datasets/titan/__init__.py

```diff
@@ -331,6 +330,24 @@ class TitanSettings:
     file_format: Literal["npy", "grib"] = "grib"
 
 
+#############################################################
+#                      BROWSE DATASET                       #
+#############################################################
+
+
+def browse_titan(period: Period):
+    """
+    Create a list of arguments used to instantiate Sample. This function indicates how to 
+    run through / browse Titan.    
```

Comment:
Similarly here, maybe just "browse" is ok (no mention of titan imho) ; or even more explicit ? "run_through_timestamps" / "run_through_dates" / "chronological_description" ?

Thinking about documentation :  we both see what you mean in the docstring but this might not be obvious for others. I would like to have another's point of view.

Reply:
+ (sorry) add type hint for output here too

Reply:
run_through_timestamps is better you are right, I m leaving the comment open for the documentation question